.github/pull_request_template.md

```diff
@@ -0,0 +1,12 @@
+## What is being delivered?
+
+Describe here all the changes that will be made and what the expected result is. If an issue was created you should also
+reference it!
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent. -->
**suggestion (non-blocking):** On Github, we can use [HTML comment](https://github.github.com/gfm/#example-148) style (`<!-- -->`) to comment blocks. What do you think of changing those lines (and others too) to:

```suggestion
<!-- Describe here all the changes that will be made and what the expected result is. If an issue was created you should also
reference it! -->
```

<!-- Can you see this? If so, give us a 👊  -->